libraries/chain/include/eosio/chain/global_property_object.hpp

```diff
@@ -153,16 +153,23 @@ namespace eosio { namespace chain {
 
    inline snapshot_global_property_object::extension_t get_gpo_extension(const global_property_object& gpo) {
       return snapshot_global_property_object::extension_v0{
-          gpo.proposed_security_group_block_num, gpo.proposed_security_group_participants, gpo.transaction_hooks};
+          gpo.proposed_security_group_block_num,
+          {gpo.proposed_security_group_participants.begin(), gpo.proposed_security_group_participants.end()},
+          {gpo.transaction_hooks.begin(), gpo.transaction_hooks.end()}};
```

Comment:
same here, why not to utilize copy-constructor here for `transaction_hooks` and `proposed_security_group_participants`?

Reply:
see above